packages/nextjs/app/debug/_components/contract/utilsDisplay.tsx

```diff
@@ -19,7 +19,7 @@ type ResultFontSize = "sm" | "base" | "xs" | "lg" | "xl" | "2xl" | "3xl";
 export const displayTxResult = (
   displayContent: DisplayContent | DisplayContent[],
   fontSize: ResultFontSize = "base",
-): string | ReactElement | number => {
+): string | ReactElement<any> | number => {
```

Comment:
Umm why do we need this? If remove the `<any>` and run 

`yarn next:check-types` or `yarn next:lint` 

nothing breaks, was this added by codemode?